themes/bootstrap3/templates/combined/results-list.phtml

```diff
@@ -28,15 +28,29 @@
       <? foreach (($top = $results->getRecommendations('top')) as $current): ?>
         <?=$this->recommend($current)?>
       <? endforeach; ?>
-      <?=$this->transEsc("Showing")?>
-      <strong><?=$this->localizedNumber($results->getStartRecord())?></strong> - <strong><?=$this->localizedNumber($results->getEndRecord())?></strong>
-      <? if (!isset($view->skipTotalCount)): ?>
-        <?=$this->transEsc('of')?> <strong><?=$this->localizedNumber($recordTotal)?></strong>
+      <?
+        $transParams = [
+          '%%lookfor%%' => $this->escapeHtml($lookfor),
+          '%%start%%' => $this->localizedNumber($results->getStartRecord()),
+          '%%end%%'   => $this->localizedNumber($results->getEndRecord()),
+          '%%total%%' => $this->localizedNumber($recordTotal)
+        ];
+        $showingResults = '';
+      ?>
+      <? if (!isset($this->skipTotalCount)): ?>
+        <? $showingResults = $this->translate('showing_results_of_html', $transParams); ?>
+      <? else: ?>
+        <? $showingResults = $this->translate('showing_results_html', $transParams); ?>
       <? endif; ?>
       <? if (isset($view->overrideSearchHeading)): ?>
+        <?=$showingResults ?>
         <?=$view->overrideSearchHeading?>
       <? elseif ($params->getSearchType() == 'basic'): ?>
-        <?=$this->transEsc('for search')?>: <strong>'<?=$this->escapeHtml($lookfor)?>'</strong>,
+        <? if (!isset($this->skipTotalCount)): ?>
```

Comment:
Why not just do another `<?=$showingResults ?>` here instead of this whole if/else to avoid duplicate logic? I think the variable will already be set to the correct value at this point based on lines 40-43. Sorry if I'm missing something....